client/dev/index.jsx

```diff
@@ -1,22 +1,34 @@
 import React from "react";
 import ReactDOM from "react-dom";
- 
-var HelloWorld = React.createClass({
-  render: function() {
-    return (
-      <p>Hello, {this.props.greetTarget}!</p>
-    );
+import NavBar from "./navbar.jsx";
+
+var navdata = [
```

Comment:
Use `const` instead of `var`. It may also be a good idea to look into [Immutable.js](https://facebook.github.io/immutable-js/) to allow React to shallowly compare data on changes rather than deeply. You would define this as an Immutable data structure rather than as the native array.